web/src/components/PageFooter.tsx

```diff
@@ -48,17 +46,16 @@ export const PageFooter = (): ReactElement => {
           <hr className="margin-y-2" />
           <div className={`${isLargeScreen ? "" : "flex flex-column flex-align-center"}`}>
             <span className="padding-right-1 text-center">
-              <UnStyledButton
-                onClick={(): void => openInNewTab(Config.footer.officeLink)}
-                className="text-white text-center"
-                ariaLabel="Made with love by the Office of Innovation"
-                style="footerLink"
-                isUnderline={true}
+              <a
```

Comment:
[boulder] Can you provide context as to why we're removing this? This recent update was done to make "Made with ❤️..." accessible.

Reply:
an anchor link should be used since it's an external link